frontends/election-manager/src/screens/write_ins_screen.tsx

```diff
@@ -137,7 +138,7 @@ export function WriteInsScreen(): JSX.Element {
       );
     }
 
-    if (!castVoteRecordFiles.wereAdded) {
+    if (cvrFilesQuery.data?.length === 0) {
```

Comment:
This is a short-hand way to handle loading/error. I've experimented with it a bit and don't mind it, but I think I'm leaning more towards explicit handling (or at least a consistent pattern with other uses of the query)